### PR TITLE
Adding is_legendary and is_mythical to pokemon_species

### PR DESCRIPTION
**Pull request by nairraghav**
Adding the is_legendary, is_mythical fields.

It looked like the order was using the wrong field as well

### CODE REVIEW

File: pokemon_v2/migrations/0007_auto_20200809_2028.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        ("pokemon_v2", "0006_pokemonspecies_is_legendary"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="pokemonspecies",
+            name="is_mythical",
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
```
Comment (phalt) [inline pokemon_v2/migrations/0007_auto_20200809_2028.py]: Please can you squash these migrations into a single file? Will make it easier to manage in the future.

Comment (nairraghav) [reply pokemon_v2/migrations/0007_auto_20200809_2028.py]: Good call, done!

**Comment by Naramsim** [comment]
Hello @nairraghav, could you please only map `is_legendary` and `is_mythical`? So to not have a new column on the CSV file?

In future maybe we will add the `is_ultra_beast` flag, but as for now, I think it's better to go on with the two attributes we have in the current CSV data.

**Comment by nairraghav** [comment]
Sure thing, I will hopefully get to that tonight

**Comment by nairraghav** [comment]
This should be resolved now

**Comment by Naramsim** [comment]
Merging tomorrow

**Comment by Naramsim** [comment]
@nairraghav, do you happen to know if in the latest gen8 there are some mythical/legendary pokemon?

**Comment by pokeapi-machine-user** [comment]
A [PokeAPI/api-data](https://github.com/PokeAPI/api-data) refresh has started. In 45 minutes the staging branch of [PokeAPI/api-data](https://github.com/PokeAPI/api-data/tree/staging) will be pushed with the new generated data. <br><br> The staging branch will be deployed in our [staging environment](https://staging.pokeapi.co/api/v2/) and you will be able to review the entire API. <br><br> A Pull Request ([master](https://github.com/PokeAPI/api-data/tree/master)<-[staging](https://github.com/PokeAPI/api-data/tree/staging)) will be also created at [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) and assigned to the PokeAPI Core team to be reviewed. If approved and merged new data will soon be available worldwide at [pokeapi.co](https://pokeapi.co/api/v2/).

**Comment by pokeapi-machine-user** [comment]
The updater script has finished its job and has now opened a Pull Request towards [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) with the updated data. <br><br> You can see the Pull Request deployed at our [staging environment](https://staging.pokeapi.co/api/v2/) when [CircleCI deploy](https://app.circleci.com/pipelines/github/PokeAPI/deploy) will be finished (_check the started time of the last build_).

**Comment by nairraghav** [comment]
@Naramsim For gen 8, I believe the legendaries are:

- Zacian
- Zamazenta
- Eternatus
- Kubfu
- Urshifu

I believe the only Mythical is:
- Zarude

**Pull request merged**